config/routes.js

```diff
@@ -146,6 +146,7 @@ module.exports.routes = {
   'GET    /noo/logout':                                   'SessionController.destroy',
 
   'POST    /noo/pn/add-device':                           'PushNotificationController.addDevice',
+  'POST    /noo/pn/disable-device':                       'PushNotificationController.disableDevice',  
```

Comment:
The more "RESTful" way of setting this up, following what I've been trying to do with the code so far, would be to have `POST /noo/device => DeviceController.create` and `DELETE /noo/device/:token => DeviceController.destroy`.